### PR TITLE
support of proxy in front of UV and CAS

### PR DESCRIPTION
**Pull request by skrchnavy**
This solves issues with redirection to CAS when UV is behind the proxy.
It checks presence of proxy headers.

### CODE REVIEW

**Comment by skrchnavy** [comment]
when feature finished, it have to be integrated (cherry pick) also into hotfix 2.0.2

File: frontend/src/main/webapp/WEB-INF/config.sample.properties [hunk 1]
```diff
@@ -114,6 +114,9 @@ admin.permission=administrator
 #CAS logout URL
 #cas.logout.url=https://192.168.1.8:8443/cas2/logout?service=http%3A%2F%2F192.168.7.42%3A18080%2Funifiedviews%2F
 
+#in case this property is true you can define relative address in property cas.security.check.url
+#behindProxy=false
```
Comment (tomas-knap) [inline frontend/src/main/webapp/WEB-INF/config.sample.properties]: Shouldn't we call it rather cas.behindProxy? Or do you foresee any use of such property in non-CAS use case?

**Comment by tomas-knap** [comment]
Minor comment above, otherwise OK from my site.

**Comment by skrchnavy** [comment]
@peterklimo nbefore finishig hotfix 2.0.2, please update property name.
Same is also valid for packages.

**Comment by skrchnavy** [comment]
released in 2.0.2 as cherrypick, then this branch merged to develop (finalisation)

**Pull request merged**